packages/tokens/src/tokens.ts

```diff
@@ -24,14 +24,28 @@ export const tokens = {
     lg: { value: '12px' },
   },
   sizes: {
-    centeredPageFullWidth: { value: '500px' },
```

Comment:
Removing `centeredPageFullWidth` is a breaking change so I need to update to `0.0.10` in the containers branch